src/main/resources/application.yml

```diff
@@ -4,32 +4,52 @@ spring:
       client:
         provider:
           keycloak:
-            issuer-uri: http://localhost:8080/auth/realms/TeamsFinder
+            issuer-uri: http://localhost:8080/auth/realms/hackathon-organizer
         registration:
           keycloak-client:
             provider: keycloak
-            client-id: teamsfinder
-            client-secret: 13c0d69f-5c7a-4e48-aaf2-eb89d16aa6d2
+            client-id: hackathon-organizer-client
+            client-secret: 0RSKBHpuEvfpvXkx5NKJclJRVGYlkf6J
             authorization-grant-type: authorization_code
             redirect-uri: "{baseUrl}/login/oauth2/code/keycloak"
   cloud:
     gateway:
+      globalcors:
+        default-filters:
+          - DedupeResponseHeader=Access-Control-Allow-Origin Access-Control-Allow-Credentials
+        corsConfigurations:
+          '[/**]':
+            allowedOrigins: "http://localhost:4200"
+            allowedHeaders: "*"
+            allowedMethods: "*"
       routes:
         - id: userWriteModule
-          uri: lb://USER-WRITE-SERVICE
+          uri: lb://user-write-service
           predicates:
-            - Path=/**
+            - Path=/api/v1/write/users/**
```

Comment:
Tego prefixu api/v1 nie ma w user servicach, musimy to ustalic czy w koncu uzywamy czy nie, daj znac czy mam poprawic w swoich serwisach